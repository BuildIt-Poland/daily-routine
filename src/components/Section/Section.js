import styled from 'styled-components';
import { spacingMedium } from '../../styles/designTokens';

const Section = styled.section`
  padding: ${spacingMedium};
`;

export default Section;
