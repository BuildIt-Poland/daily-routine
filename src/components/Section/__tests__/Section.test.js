import React from 'react';
import { create } from 'react-test-renderer';

import Section from '../Section';

describe('COMPONENT - Section', () => {
  it('renders correctly', () => {
    const component = create(<Section />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
