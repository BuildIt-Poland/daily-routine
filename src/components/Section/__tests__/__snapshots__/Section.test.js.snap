// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Section renders correctly 1`] = `
.c0 {
  position: relative;
  padding: 0;
  padding-top: 2rem;
}

<section
  className="c0"
/>
`;
